apps/marsha/templates/services/app/cronjob_check_live_state.yml.j2

```diff
@@ -25,6 +25,10 @@ spec:
             version: "{{ marsha_image_tag }}"
             deployment_stamp: "{{ deployment_stamp }}"
         spec:
+{% if marsha_image_pull_secret_name is defined and marsha_image_pull_secret_name is not none %}
```

Comment:
Missing indentation ? It should be on the same level than the containers definition.